pkg/cloud_provider/file/file.go

```diff
@@ -31,7 +32,7 @@ import (
 	"google.golang.org/api/option"
 	"google.golang.org/grpc/codes"
 	"k8s.io/apimachinery/pkg/util/wait"
-	"k8s.io/klog"
+	"k8s.io/klog/v2"
```

Comment:
hmm I wonder if moving to v2 will affect other logging as well. V1 is deprecated so we SHOULD be using v2 but is v2 backwards compatible? @msau42 do you know?

Reply:
moving this comment from #380 because it's closed

Reply:
The Verbose type changed which caused klog to rev to v2: https://github.com/kubernetes/klog/commit/cd60aa438f9c750641d5feacd2999e58df8af28f

This may also be important if we are supporting both klog v1 AND v2 AND glog? We probably want to convert it all https://github.com/kubernetes/klog#coexisting-with-klogv2

Reply:
IIUC: anywhere we use klog.V() might be broken because klog v2 changed this to return a struct instead of a bool? Tests have been passing so sounds like mostly we don't use that

Reply:
I did not see we use the return value of klog.V() in the code base so it should not impact anything. Do you have know if there is any place we might be doing that?

Reply:
yeah that's my understanding. should be the same. code search don't show up anything

Reply:
Also double check that the various types of logs, info, warn, error, etc are still printed as expected. There was some incompatibility between command line arguments between glog, klog and klog v2

Reply:
@sunnylovestiramisu can you double check as suggested

Reply:
See https://github.com/kubernetes-csi/external-provisioner/pull/518 for an example issue that can happen if you have multiple klog versions

Reply:
From the test logs: http://shortn/_e4Bnk4egqd

```
klog.Errorf() in file.go:318 Failed to get...
klog.Infof() in controller.go:260 CreateVolume succeeded...
```

From pod logs: http://shortn/_ONYnRuhl5B
```
klog.Warningf() in mount_helper_common.go:133] Warning...
```